docs/implementation-status.md

```diff
@@ -0,0 +1,245 @@
+### Support = partial
+
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_get`                               | yes | n\a |
+| `prif_get_indirect`                      | yes | n\a |
+| `prif_put`                               | yes | n\a |
+| `prif_put_indirect`                      | yes | n\a |
+| `prif_put_with_notify`                   | no  | n\a |
+| `prif_put_with_notify_indirect`          | no  | n\a |
+| `prif_put_indirect_with_notify`          | no  | n\a |
+| `prif_put_indirect_with_notify_indirect` | no  | n\a |
+
+---
+
+## Strided Coarray Access
+### Support = no
+
+<!---
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_get_strided`                               | no  | n\a |
+| `prif_get_strided_indirect`                      | no  | n\a |
+| `prif_put_strided`                               | no  | n\a |
+| `prif_put_strided_indirect`                      | no  | n\a |
+| `prif_put_strided_with_notify`                   | no  | n\a |
+| `prif_put_strided_with_notify_indirect`          | no  | n\a |
+| `prif_put_strided_indirect_with_notify`          | no  | n\a |
+| `prif_put_strided_indirect_with_notify_indirect` | no  | n\a |
+-->
+
+---
+
+## SYNC Statements
+### Support = partial ( only support for `prif_sync_all`)
+
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_sync_memory` | no  | n\a |
+| `prif_sync_all`    | yes | n\a |
+| `prif_sync_images` | no  | n\a |
+| `prif_sync_team`   | no  | n\a |
+
+---
+
+## Locks and Unlocks
+### Support = no
+
+<!---
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_lock`            | no  | n\a |
+| `prif_lock_indirect`   | no  | n\a |
+| `prif_unlock`          | no  | n\a |
+| `prif_unlock_indirect` | no  | n\a |
+-->
+
+---
+
+## Critical
+### Support = no
+
+<!---
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_critical`     | no  | n\a |
+| `prif_end_critical` | no  | n\a |
+-->
+
+---
+
+## Events and Notifications
+### Support = no
+
+<!---
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_event_post`          | no  | n\a |
+| `prif_event_post_indirect` | no  | n\a |
+| `prif_event_wait`          | no  | n\a |
+| `prif_event_query`         | no  | n\a |
+| `prif_notify_wait`         | no  | n\a |
+-->
+
+---
+
+## Teams
+### Support = partial (No support for `prif_get_team` and `prif_team_number`)
+
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_form_team`   | yes | n\a |
+| `prif_get_team`    | no  | n\a |
+| `prif_team_number` | no  | n\a |
+| `prif_change_team` | yes | n\a |
+| `prif_end_team`    | yes | n\a |
+
+---
+
+## Collectives
+### Support = partial (...)
+
+| Procedure | Status | Notes |
+|-----------|--------|-------|
+| `prif_co_broadcast` | ... | ... |
+| `prif_co_max`       | ... | ... |
+| `prif_co_min`       | ... | ... |
+| `prif_co_sum`       | ... | ... |
+| `prif_co_reduce`    | ... | ... |
```

Comment:
```suggestion
| `prif_co_broadcast` | partial | no support for derived types with `allocatable` components |
| `prif_co_max`       | partial | only supports 32-bit and 64-bit numeric types |
| `prif_co_min`       | partial | only supports 32-bit and 64-bit numeric types |
| `prif_co_sum`       | partial | only supports 32-bit and 64-bit numeric types |
| `prif_co_reduce`    | partial | only supports intrinsic types (no support for derived types) |
```

Reply:
`prif_co_{min,max}` have an implementation for character strings, but @rouson and I have just discovered that it's "fragile" (issue #132) such that we might not want to claim support